src/main/java/userInterface/Validity.java

```diff
@@ -0,0 +1,69 @@
+package userInterface;
+
+public class Validity {
+    private String description;
+    public boolean isValid;
```

Comment:
It is good practise to make this variable private within this class so that it may not be accidentally changed by other parts of the large program. You can access it with a public method. This will limit access to the variable and prevent unintentional bugs.